user_storage_postgresql.py

```diff
@@ -1,50 +1,64 @@
+import pg8000
 from user import User
 from datetime import datetime, timezone
-from postgresql_singleton import PostgreSQLSingleton
+from bot.config_reader import env_config
 from typing import List
 
 
+def get_connection():
+    return pg8000.connect(
+        user=env_config.postgresql_username,
+        password=env_config.postgresql_password.get_secret_value(),
+        host=env_config.postgresql_hostname,
+        port=env_config.postgresql_port,
+        database=env_config.postgresql_database,
+    )
```

Comment:
При такое реализации я не уверен, что произойдет закрытие соединения с БД. Даже если далее писать `with get_connection()...`

Вот так будет правильно:

```suggestion
    with pg8000.connect(
        user=env_config.postgresql_username,
        password=env_config.postgresql_password.get_secret_value(),
        host=env_config.postgresql_hostname,
        port=env_config.postgresql_port,
        database=env_config.postgresql_database,
    ) as connection:
        yield connection
```